tests/testthat/test-qgis-result.R

```diff
@@ -26,4 +26,7 @@ test_that("qgis_result_*() functions work", {
   expect_true(all(c("URL", "OUTPUT") %in% names(qgis_result_args(result))))
   expect_is(qgis_result_stderr(result), "character")
   expect_is(qgis_result_stdout(result), "character")
+  expect_error(qgis_result_single(result, "numeric"), "zero outputs of type")
+  expect_identical(qgis_result_single(result, "qgis_outputFile"),
+                   result$OUTPUT)
```

Comment:
With pleasure. Correct me if I'm wrong; I understand it as follows - i.e. how I applied it in a3f5001:

**Wrong**

```r
  expect_identical(qgis_result_single(result, "qgis_outputFile"),
                   result$OUTPUT)
```

The requirement being: separate lines for arguments and for `)` when, and only when, arguments are splitted over multiple lines.

**Right**

```r
  expect_identical(
    qgis_result_single(result, "qgis_outputFile"),
    result$OUTPUT
    )

```

**Also right (?), but unneeded since this fits in one line**

```r
  expect_identical(
    qgis_result_single(
      result, 
      "qgis_outputFile"
      ),
    result$OUTPUT
    )
```